18F/identity-idp
# Use Webpack manifest to associate asset references per pack

**Why**: Since our current approach to list assets required per pack is not scalable, and does not account for packs other than `document-capture`, nor would it support assets deeply referenced via shareable packages (e.g. `@18f/identity-components`).

**What**:

- Implements a Webpack plugin similar to our localization plugin, which scans for calls to `getAssetPath`
- Associates those paths with the entrypoint(s) they are included within
- When those entrypoints are printed, the paths are mapped using Rails `asset_path`, and assigned to a window global
- The front-end calls to `getAssetPath` pull the mapped path from the window global

**Testing Instructions:**

Ensure there are no regressions to the use of images in the document capture step, e.g. the "warning" icon after a failed submission, or the ID card animated SVG loading interstitial.
